### DISCUSSION TITLE
Nina Controlling Camera not working

### DISCUSSION THREAD

**Discussion by farn32** [mworion/MountWizzard4#334] [Q&A] [answered]
Version Mountwizzard v3.3.2 
I have trouble setting the camera controlled bi nina in MW. I can attach the camera without a problem either in NINA or in MW. It's a Moravian C3 26000 CMOS camera. Nina is 64 Bit and i use the 64 Bit driver. MW, resp. Python is also 64 Bit.
When python Version is 64 Bit is the MW version also 64 Bit?
Maybe any hints?

Thanks alot.

**Comment by mworion** [comment]
Hi,
try to give some hints:
- In NINA the server function is enabled? In actual v2 it's under system, in v3 nightly it's a separate plugin
- if you choose in MW4 NINA does the selector turn green? if not this hints to server not enabled.
- Both programs should be on the same network
- If MW4 set to NINA controlled, you have to connect the camera in NINA to be used in MW4
Better analyze could be done if you post a current MW4 log file to see what might be wrong.
Michel

**Reply by farn32** [reply]
Hi Michel
Thank you very much for your help. I didn't now the thing with the server in Nina. Everything is OK now and it works.
Many thanks.
All the best, Fritz

**Comment by Maxastro59** [comment]
Hello, I'm new in MW4 and have the same issue with NINA chooser. I think there is a problem with server connection but seems to be connected to the serve while I do not know how to do it on MW4 (see pic). Also their MW Virtual Keypad does not connecto to the mount (GM2000 QCI, not HP). Thanks Max

<img width="1844" alt="Screenshot 2023-12-22 alle 11 54 03" src="https://github.com/mworion/MountWizzard4/assets/154596811/e0364da4-f17e-46d1-9208-c8003a0b72ee">

**Comment by mworion** [comment, marked as answer]
Hi Max, here is the setting in NINA:
![Bildschirmfoto 2023-12-22 um 12 04 30](https://github.com/mworion/MountWizzard4/assets/6632980/477c613a-c25c-43a3-b864-4bfe3e80452a)

NINA and MW4 have to be on the same computer.

Michel

**Comment by Maxastro59** [comment]
Hi, many thanks for your quick reply! Yes my error is due to the fact that I use NINA on two PCs for two different setups and the server was activated on the other. In fact, NINA now connects to MW without problems. In addition to the question regarding the Virtual Keypad that I have already asked you, I would like to know what is the best way to build a model with my GM2000 QCI which is not the latest HPS version. At a time when the model was built manually by centering the stars (and not the points), it was necessary to put the mount in the home state (Reset at Home), build the basic model with three stars and then do the Refine. The same procedure was implemented with the first model makers (that of the late Frejwall for example): 3 points + refine. Now with MW4 I would like to know:
1- if it is necessary to place the mount in the Home position before starting to build the model: this would be a problem for me because my observatory is managed remotely and is located 150 km from home.
2 - if there is a need to memorize the three basic points first or you can directly build the model in a single solution.
3 - Whether it is better to make one from the beginning or add points to an existing one that works quite well
4 - if by building a new model it is saved without deleting those already present.
Thanks for your support.

**Comment by mworion** [comment]
Hi Max,
unfortunately I do not have any experiences with the QCI versions of the mount. I own a GM1000HPS. So there might be some glitches. But please report them as I would add the in the docs for other users. Most probably they are not solvable.

To you questions:

1 - Honestly I do not know. For the GM1000HPS there is no need for it.
2 - The procedure described by 10micron is an iterative one. Honestly, there is no difference between the first 3 points and all others. Also the mint does not store or treat them differently. The remark about the 3 first points is that once you succeed with three point, you will get a first model with first corrections in place to move on. MW4 does it all at once, so gathering all the data from all points and "program" the model in a batch procedure.
3 - as said MW4 does not support the iterative way, just batch.
4 - MW4 backs up the actual model before starting a new one under the name "Backup". Please be reminded, if you start modeling multiple times for any reason, this backup model will be overwritten each attempt. If you would like to store the model (wich I do normally to not mix up everything during testing) please save the actual model under a name of you choice.

Hope this helps.
Michel

**Comment by Maxastro59** [comment]
Many thanks and Best Whises
